test/suites/remote.sh

```diff
@@ -79,7 +79,7 @@ test_remote_url_with_token() {
   token="$(lxc config trust list-tokens -f json | jq '.[].Token')"
 
   # create new certificate
-  openssl req -x509 -newkey rsa:2048 -keyout "${TEST_DIR}/token-client.key" -nodes -out "${TEST_DIR}/token-client.crt" -subj "/CN=lxd.local"
+  openssl req -x509 -newkey ec -pkeyopt ec_paramgen_curve:secp384r1 -sha384 -keyout "${TEST_DIR}/token-client.key" -nodes -out "${TEST_DIR}/token-client.crt" -days 1 -subj "/CN=lxd.local"
```

Comment:
It'd be good to add a comment here and below (or combine into a function ) explaining how these arguments are chosen for this test. (I'm not clear what benefit this is giving the tests)

Reply:
Apart from the short lived part, which makes sense for a test.

Reply:
ECDSA certs are what LXD uses by default and what we've been suggesting in our doc (for the metrics cert). Those are smaller and for the most part faster to use (less CPU intensive). RSA 2048 was fine, I just changed it for consistency. Would `# Create a new short lived ECDSA certificate.` work?

Reply:
Yeah, is there a specific decision around `sha384` and the particular curve?

Reply:
That's also to match LXD's preferred hash and curve. Long ago, LXD used RSA/4096 so I would assume the intention was to offer more than ~128 bits of security (https://www.keylength.com/en/4/) without resorting to overly big (and slow) RSA keys.

Reply:
Great, that makes sense, thanks. Lets put that into a comment into a shared function we can use in our tests, then at least we can show justification for those openssl arguments. It'll make it easier in the long run and help others understand why a decision has been made. Thanks

Reply:
Done